src/CarbonAware.WebApi/src/README.md

```diff
@@ -176,7 +176,7 @@ If time period is not provided, it retrieves all the data until the current time
 
 EG
 ```
-https://<server_name>/forecasts/current?location=northeurope&startTime=2022-07-19 14:00&endTime=2022-07-20&windowSize=10
+https://<server_name>/emissions/forecasts/current?location=northeurope&startTime=2022-07-19 14:00&endTime=2022-07-20&windowSize=10
```

Comment:
Line 162 above should say ### GET emissions/forecasts/current
Line 12 should say [GET emissions/forecasts/current](#get-forecastscurrent)

(Cannot comment directly on those lines)